src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py

```diff
@@ -70,6 +71,10 @@ def __init__(self, config):
         # set resource control
         self.resourceControl = ResourceControl(config=self.config)
 
+        # get CRIC wrapper instance
+        self.cricPattern = ".*"  # all sites
```

Comment:
there is no need to create self data with static value since it never changed. Unless it shown otherwise I suggest to move value `.*` directly into `updatePNNMap` function and put it into  function call where the comment will be more appropriate.

Reply:
I see pros and cons in each implementation. With this, we have a clear definition at the initialization of the object about which parameters are used within this module/thread. Creating a constant inside the function is fine as well. I will update this code soon.